### PR TITLE
Remove all mentions of fireEvent.

### LINKED ISSUE #22: dash-daq breaks with the newest version of dash (0.36.0)
Due to events being removed from Dash, `dash-daq` fails during installation.

### PR DESCRIPTION
**Pull request by shammamah**
Closes #22. Ensures compatibility with the newest version of Dash (`0.36.0`).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,19 +5,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
-### Removed
+## [0.1.0] - 2019-01-28
 
 ### Fixed
+* Fixed incompatibility issue with Dash `0.36.0`. 
 
-### Security
+### Removed 
+* Removed unused headers in CHANGELOG. 
+* Removed all mentioned of `fireEvent` and anything else that used Dash events (which are now deprecated).
```
Comment (Marc-Andre-Rivet) [inline CHANGELOG.md]: Would add a mention to use n_clicks / timestamp instead

File: src/components/__tests__/ToggleSwitch.test.js [hunk 2]
```diff
@@ -45,16 +35,7 @@ describe('Toggle Switch', () => {
...
-    const eventReciever = sinon.spy();
-    const component = mount(<ToggleSwitch disabled={true} fireEvent={eventReciever} />);
-
-    component.find('button').simulate('click');
-
-    expect(eventReciever.calledOnce).toBeFalsy();
-  });
-
-  it('handles absent setProps and fireEvent callbacks', () => {
+  it('handles absent setProps callbacks', () => {
```
Comment (Marc-Andre-Rivet) [inline src/components/__tests__/ToggleSwitch.test.js]: Same-ish test for 3 comps with very different names :)

does not explode when setprops is not passed
handles unbound setProps prop
handles absent setProps callbacks

File: src/components/__tests__/BooleanSwitch.test.js [hunk 3]
```diff
@@ -25,16 +25,6 @@ describe('Boolean Switch', () => {
...
-  it('fires click event on click', () => {
-    const eventReciever = sinon.spy();
-    const component = mount(<BooleanSwitch fireEvent={eventReciever} />);
-
-    component.find('button').simulate('click');
-
-    expect(eventReciever.calledOnce).toBeTruthy();
-    expect(eventReciever.getCall(0).args[0].event).toBe('click');
-  });
-
```
[the remark this answers is not in the document]
Comment (shammamah) [reply src/components/__tests__/BooleanSwitch.test.js]: This is just a switch component, so it doesn't really have an `n_clicks` property -- just a Boolean `on` property. The file still has this: 

```
    expect(setProps.getCall(0).args[0].on).toBeTruthy();
```

Or should I add something else?

File: CHANGELOG.md (hunk 1 above)
Comment (Marc-Andre-Rivet) [reply CHANGELOG.md]: Or `value` in the case of the Slider, I think?

File: src/components/__tests__/BooleanSwitch.test.js [hunk 4]
```diff
@@ -47,14 +37,14 @@ describe('Boolean Switch', () => {
 
   it('does not fire click event when disabled and clicked', () => {
     const eventReciever = sinon.spy();
-    const component = mount(<BooleanSwitch disabled={true} fireEvent={eventReciever} />);
+    const component = mount(<BooleanSwitch disabled={true} />);
```
Comment (alexcjohnson) [inline src/components/__tests__/BooleanSwitch.test.js]: `eventReceiver` isn't referenced by anything now, so this test isn't doing anything... but this is the only mention of `disabled` in the suite, is there an equivalent test we can write that `on` is never modified?

File: src/components/__tests__/BooleanSwitch.test.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Marc-Andre-Rivet) [reply src/components/__tests__/BooleanSwitch.test.js]: Sounds good.

File: CHANGELOG.md (hunk 1 above)
Comment (alexcjohnson) [reply CHANGELOG.md]: In fact Dash events are not deprecated - that would mean they're still there, just discouraged. They've been removed!

File: src/components/__tests__/BooleanSwitch.test.js (hunk 4 above)
Comment (shammamah) [reply src/components/__tests__/BooleanSwitch.test.js]: Here's an idea for that: 
```
it('does not change props when disabled and clicked', () => {
    const setProps = sinon.spy(); 
    const component = mount(<BooleanSwitch setProps={setProps} on={false} disabled={true} />);
    expect(component.state('on')).toBeFalsy(); 
    component.find('button').simulate('click'); 
    expect(component.state('on')).toBeFalsy(); 
}); 
```
Should we also include a test for the opposite case (button is on and disabled)?

Comment (alexcjohnson) [reply src/components/__tests__/BooleanSwitch.test.js]: That looks great. I don't think it's necessary to test both cases.

**Pull request merged**